cmd/gitpods/dev_setup.go

```diff
@@ -49,6 +53,27 @@ func ActionDevSetup(c *cli.Context) error {
 	return nil
 }
 
+func setupPostgres() error {
+	docker, err := exec.LookPath("docker")
+	if err != nil {
+		return err
+	}
+
+	args := []string{
+		"run", "-d",
+		"--name", "gitpods-postgres",
+		"-p", "5432:5432",
```

Comment:
That's just the `gitpods dev --setup` which gets you up and running more quickly. You can always start other containers and provide `gitpods dev --database-datasource` with a different connection.